web/src/pages/map/doc.vue

```diff
@@ -61,6 +61,15 @@
         <p v-html="example"></p>
       </template>
 
+      <h6>👁️ <translate>Hide markers</translate></h6>
+      <p><translate-slot>
+        <span translate>Do not want to see {title} markers?</span>
+        <template v-slot:title>
+          <i>{{ title }}</i>
+        </template>
+      </translate-slot></p>
+      <b-button size="sm" v-on:click.stop.prevent="$emit('hide-item-markers', item)">Hide from map</b-button>
```

Comment:
Please, translate the content of the button.

Reply:
It is just lacking a `translate` attribute, or something more must be done? I don't know how translations are working in Osmose frontend.

Reply:
`<translate>foo</translate>` or `<translate-slot>`